quesma/queryparser/query_parser.go

```diff
@@ -217,36 +217,6 @@ func (cw *ClickhouseQueryTranslator) ParseHighlighter(queryMap QueryMap) model.H
 	return highlighter
 }
 
-func (cw *ClickhouseQueryTranslator) ParseQueryAsyncSearch(queryAsJson string) (model.SimpleQuery, model.HitsCountInfo, model.Highlighter) {
```

Comment:
Was used 3 times: in 3 different tests. Relic from before unification of async/normal search.